client/cjdroute2.c

```diff
@@ -173,6 +177,15 @@ static int genconf(struct Random* rand, bool eth)
            "                \"connectTo\":\n"
            "                {\n"
            "                    // Add connection credentials here to join the network\n"
+           "                    // If you have several, don't forget the separating commas\n"
+           "                    // They should look like:\n"
+           "                    // \"ipv4 address:port\": {\n"
+           "                    //     \"password\": \"password to connect with\",\n"
+           "                    //     \"publicKey\": \"remote node key.k\",\n"
+           "                    //     // And optionally:\n"
+           "                    //     \"ipv6\": \"derived from key with publictoip6\",\n"
```

Comment:
why "ipv6" ?   It really doesn't do anything and it's tempting fate in case in the future we decide to use it for something.